File: week2/hw4.js
function twoSum(nums, target) {
  let result = [];
  for (i = 0; i < nums.length; i++) {
    for (j = i + 1; j < nums.length; j++) {
      let sum = nums[i] + nums[j]
      if (sum === target) {
        result.push(i, j)
      }
    }
  } return result
}

let result = twoSum([2, 11, 7, 15], 13);
console.log(result);

File: week2/hw4.py
def twoSum(nums, target):
    result = []
    for i in range(0, len(nums)):
        for j in range(i+1, len(nums)):
            if nums[i] + nums[j] == target:
                result = [i, j]
    return result


result = twoSum([2, 11, 7, 15], 9)
print(result)
